CHANGELOG.md

```diff
@@ -9,8 +9,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+- Added support for python 3.11
+  ([#72](https://github.com/dsgibbons/shap/pull/72) by @connortann)
 - Added `n_points` parameter to all functions in `shap.datasets`
-  ([dsgibbons#39](https://github.com/dsgibbons/shap/pull/39) by @thatlittleboy).
+  ([#39](https://github.com/dsgibbons/shap/pull/39) by @thatlittleboy).
```

Comment:
Sorry but I don't really agree with this change. 
Clarity is more important than conciseness in a CHANGELOG imo.

Reply:
I agree that clarity is important than conciseness in a changelog. What do you find unclear about this proposal?

I actually think the current "fork owner" prefixes are unclear and misleading, as when reading the Changelog in the current state it's easy to confuse the PR author and the fork owner. The fork owner is more prominent, mentioned first and in blue font, and given in a nonstandard shorthand `fork#number`.

I suppose it comes back to the purpose of the changelog..
- For users reading the changelog, the most important info is just "what has changed".
- It's nice to add the author, as that small bit of recognition provides an incentive to future authors to contribute PRs.
- The hyperlink to the PR URL is useful in case a user wishes to click through and see more details. The hyperlink is clear whether the changelog is viewed in raw form or as rendered mardown.

Is the "fork owner" relevant information for users in this context?

Reply:
Agreed on the purpose of the changelog. To your points:

I don't think that there will be a confusion of the fork owner for the PR contributor. The whole text is linked and in blue, so it's clear (at least to me) that it's part of the original source (i.e., a fork). The PR author is also credited at the end of the sentence.

Without specifying the fork name, we leave ourselves open to ambiguity about where a PR originates from (which I think is somewhat important in any case: it's part of documentation of the history of the project). In an extremely unlikely scenario, we have two or more entries in the changelog that refers to the same PR number `#20` (say) but are in reality pointing to different repos entirely.

But I'll rest my case here though, as I don't feel too strongly about it either way. :) I just didn't think that removing the fork owner from the text was a necessary change.

Reply:
>  part of documentation of the history of the project

That's a good point - for the internet archives! OK, let's keep it in :)

I tried rejigging it a little to be a bit more explicit, adding a short paragraph about the Community Fork to explicity state the fork is indeed merged into the main repo now.